docs/scu/dev/actions-after-upgrade.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Actions after the upgrade
+description: The document describes the after-update actions applicable for both the manual and the automatic update process using the Spryker Code Upgrader
+template: howto-guide-template
+---
+
+
+This document describes the actions that should be performed after the module update process to make sure nothing was broken.
+Those steps are applicable for both manual update process and automatic update by Spryker Code Upgrader.
+
+## Verify module files
+
+Verify all the modules' files on the project level and make sure they are compatible with the files from the vendor. Pay attention to business models, even after minor module updates they might be significantly changed since they are not a part of the module's public API. Functions might be renamed or refactored, constructors might have changed parameters, and even a whole business model might be split into several ones. The last one is a rare case, but it is still possible. So you need to check if project-level customizations are still compatible with the new module's version, and if you have new business models created on the project level and they use existing models from the vendor, they also must be reviewed.
```

Comment:
What do you mean by verifying in this case?

Reply:
I mean a developer needs to open each project-level file and check if all customizations are still compatible with the new module version. Because if they have an overwritten function its signature/behavior might be changed, or if they have a new function in an inherited business model and inside they use a function from core-level class, that function also might be changed or even deleted, etc. Everything must be verified.